my-app/lib/product-service/populateDynamoDB.ts

```diff
@@ -0,0 +1,49 @@
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb';
+import { DynamoDBDocumentClient, PutCommand } from '@aws-sdk/lib-dynamodb';
+import { uuid } from 'uuidv4';
+import productsList from './mocks/productsList.json';
+
+const client = new DynamoDBClient({ region: process.env.REGION });
+const docClient = DynamoDBDocumentClient.from(client);
+
+const productTableName = process.env.STACK_PRODUCT_DB_NAME;
+const stockTableName = process.env.STACK_PRODUCT_STOCK_DB_NAME;
+
+async function prePopulateTables() {
```

Comment:
It's not clear what is the difference between "populateDB" and "prePolutaTables" :\